src/lib/assetUtils.ts

```diff
@@ -102,3 +102,8 @@ export const getDefaultChainIDFromNetworkType = (networkType: NetworkType): stri
   if (networkType === 'cosmos') return 'noble-1';
   return undefined;
 };
+
+export const getAssetDescriptionStringKeys = (assetId: string) => ({
```

Comment:
where did this used to be?

Reply:
it never existed! It was something returned form Abacus in the resources object